addons/project/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+column_copies = {
+    'project_project': [
+        ('analytic_account_id', None, None),
```

Comment:
We shouldn't rename this column, as it requires afterwards to be renamed/copied again in `hr_timesheet`, which is useless, as the meaning of the field doesn't change, and can lead to some errors if other data is linked to this SQL column.

Reply:
it's a column_copies, not a rename

Reply:
Yes, right. As talked by chat, let's keep this in case we need it.